customcontext/texture/hybristexture.cpp

```diff
@@ -272,11 +272,10 @@ NativeBuffer *NativeBuffer::create(const QImage &image)
     return buffer;
 }
 
-HybrisTexture::HybrisTexture(NativeBuffer *buffer)
+HybrisTexture::HybrisTexture(QSharedPointer<NativeBuffer> buffer)
```

Comment:
Btw, even thou not related isn't EglGrallocTexture suffering from the same symptom.